modules/javafx.graphics/src/shims/java/javafx/scene/layout/RegionShim.java

```diff
@@ -53,12 +53,16 @@ public static double computeChildMaxAreaWidth(Region r,
                 baselineComplement, margin, height, fillHeight);
     }
 
+    public static double computeChildMinAreaHeight(Region r,
+        Node child, Insets margin) {
+    return r.computeChildMinAreaHeight(child, margin);
+}
```

Comment:
there is something wrong with indentation here.
and ... in this class

since it's a shim, I would rather auto-format the whole thing...

Reply:
I'll see what I can do